redi-connect-front/src/components/Footer.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "./Footer.scss";
+import twitter from "../assets/images/twitter.svg";
+import medium from "../assets/images/medium.svg";
+import instagram from "../assets/images/instagram.svg";
+import linkedin from "../assets/images/linkedin.svg";
+
+const Footer = () => {
+  return (
+    <div className="footer__wrapper">
+      <div className="row">
+        <div className="footer__container">
```

Comment:
There's a fair bit of nesting going on here and in other files. I wonder if we can simplify it. Can footer_wrapper and footer_container be merged?